README.md

```diff
@@ -45,26 +45,8 @@ $ docker service create \
   dockersamples/visualizer
 ```
 
-## Running on ARM
-
-[@alexellisuk](https://twitter.com/alexellisuk) has pushed an image to the Docker Hub as `alexellis2/visualizer-arm:latest` it will run the code on an ARMv6 or ARMv7 device such as the Raspberry Pi.
-
-```
-$ docker service create \
-  --name=viz \
-  --publish=8080:8080/tcp \
-  --constraint=node.role==manager \
-  --mount=type=bind,src=/var/run/docker.sock,dst=/var/run/docker.sock \
-  alexellis2/visualizer-arm:0.4
-```
-
-If you would like to build the image from source run the following command:
-
-```
-$ docker build -f Dockerfile.arm -t visualizer-arm:latest .
-```
-
-[View on Docker Hub](https://hub.docker.com/r/alexellis2/visualizer-arm/tags/)
+## Supported architectures
+The main image is based on [node:8-alpine](https://hub.docker.com/_/node/). With that it already offers support for many architectures, including ARM (so it can run on your Raspberry Pi). See below for instructions on how to run on Windows.
```

Comment:
This chapter should be more clear how to build the image for another architecture. 
I assume there will be still only the linux/amd64 image as `dockersamples/visualizer` image.

So we need at least the docker build step how to build it locally and then use the commands above to start the visualizer.